From 00d10cff555c79a0ac11f7a298a892176830c353 Mon Sep 17 00:00:00 2001
From: Zoltan Puskas <zoltan@sinustrom.info>
Date: Sat, 4 Dec 2021 13:27:23 -0800
Subject: [PATCH 2/2] Fix lint in trash-cli

Fix the import redefinition issue in list.py detected by pyflakes
---
 trashcli/list.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/trashcli/list.py b/trashcli/list.py
index 09f30de..725b5ce 100644
--- a/trashcli/list.py
+++ b/trashcli/list.py
@@ -4,7 +4,7 @@ import os
 
 from . import fstab
 from .fs import FileSystemReader, file_size
-from .fstab import volume_of, VolumesListing
+from .fstab import VolumesListing
 from .trash import (version, TrashDirReader, path_of_backup_copy, print_version,
                     maybe_parse_deletion_date, trash_dir_found,
                     trash_dir_skipped_because_parent_is_symlink,
-- 
2.33.1

